src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double height;
+    private double firstLeg;
+    private double secondLeg;
+
+    public IsoscelesTrapezoid(String color, double height, double firstLeg, double secondLeg) {
+        super(color);
+        this.height = height;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double calculateArea() {
```

Comment:
The method calculateArea() should be overridden from the AreaCalculator interface, and it's missing the @Override annotation.